### PR TITLE
feat: add flag to enable cluster load balance

### PR DESCRIPTION
**Pull request by levy5307**
add remote command to enable cluster load balance. I will add the implementation of cluster load balance in later pull requests.

### CODE REVIEW

File: src/meta/greedy_load_balancer.cpp [hunk 1]
```diff
@@ -827,6 +839,20 @@ void greedy_load_balancer::greedy_balancer(const bool balance_checker)
         }
     }
 
+    if (!_balance_cluster) {
+        app_balancer(balance_checker);
+        return;
+    }
+
+    if (!balance_checker) {
```
Comment (Smityz) [inline src/meta/greedy_load_balancer.cpp:847]: what is balance_checker?

Comment (levy5307) [reply src/meta/greedy_load_balancer.cpp:847]: It is used to get the balancer operation count

**Review by hycdong — DISMISSED** [review]

**Comment by Shuo-Jia** [comment]
why add config with gflag to update, but use remote-command?

**Comment by levy5307** [comment]
> why add config with gflag to update, but use remote-command?

I think they are both good. I change change it to using gflag if you think it is more better

**Review by Shuo-Jia — APPROVED** [review]

**Review by hycdong — APPROVED** [review]

**Pull request merged**